<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticlePresetForm</class>
 <widget class="QWidget" name="ParticlePresetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoadPreset</string>
  </property>
  <widget class="QPushButton" name="use_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Use selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="detail_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="des_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preview_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Preview uses euler method</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="preset_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
